package.json

```diff
@@ -52,5 +59,9 @@
     "ts-node": "^10.9.2",
     "typescript": "^5.6.3",
     "vitest": "^2.1.3"
+  },
+  "engines": {
+    "node": ">=18",
```

Comment:
I set Node.js to version >= 18 because the tests seem to run on 18, but it probably should be version 20.
Up to you!